src/recipe/Baked.Recipe.Service.Application/Localization/AspNetCore/AspNetCoreLocalizationExtensions.cs

```diff
@@ -15,4 +15,25 @@ public static AspNetCoreLocalizationFeature AspNetCore(this LocalizationConfigur
 
         return new(language, otherLanguages);
     }
+
+    public static Dictionary<string, string> ReadResxFileAsDictionary(string path)
+    {
+        var values = new Dictionary<string, string>();
+        using (StreamReader reader = new(path))
+        {
+            string? line = reader.ReadLine();
+            while (line != null)
+            {
+                if (!string.IsNullOrWhiteSpace(line.Trim()))
+                {
+                    var keyValue = line.Split('=', StringSplitOptions.TrimEntries);
+                    values.TryAdd(keyValue[0], keyValue[1]);
+                }
+
+                line = reader.ReadLine();
+            }
+        }
+
+        return values;
+    }
```

Comment:
this is not an extension. better to move this to where it is being used, because adding such extension to `string` would be too much. alternatively you might make this method `internal` and make it a string extension method